src/base/bittorrent/session.cpp

```diff
@@ -29,6 +29,10 @@
 
 #include "session.h"
 
+#ifdef Q_OS_WIN
```

Comment:
I would move this to the above of `const char PEER_ID[]` or `#include <cstdlib>`